### PR TITLE
Add blog post around understanding Cloud Events interactions

### PR DESCRIPTION
**Pull request by berndruecker**

### CODE REVIEW

File: community/demos.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ markdown file in `community/demos/`.
   - Blog post about Cloud Events Orchestration using an workflow engine([Zeebe](http://zeebe.io)). 
   - Instructions: You can find the [Demo Code and instructions here](https://github.com/salaboy/zeebe-cloud-events-examples).
   
+- [Gaining business visibility into processes implement by Cloud Events](https://blog.bernd-ruecker.com/gaining-visibility-into-processes-spanning-multiple-microservices-a1fc751c4c13s)
```
Comment (lance) [inline community/demos.md:10]:
```suggestion
- [Gaining business visibility into processes implemented by Cloud Events](https://blog.bernd-ruecker.com/gaining-visibility-into-processes-spanning-multiple-microservices-a1fc751c4c13)
```

Comment (berndruecker) [reply community/demos.md:10]: Great - thanks!

Comment (duglin) [inline community/demos.md:10]: I get a 404 from this url.

Comment (duglin) [reply community/demos.md:10]: https://blog.bernd-ruecker.com/gaining-visibility-into-processes-spanning-multiple-microservices-a1fc751c4c13
works though. I think the 's' at the end of the url is messing it up

**Comment by duglin** [comment]
Approved on the 6/25 call - just waiting for the typo fix before merging

File: community/demos.md (hunk 1 above)
Comment (berndruecker) [reply community/demos.md]: Good catch - sorry for that - fixed!

**Pull request merged**